src/IECoreMaya/FromMayaMeshConverter.cpp

```diff
@@ -465,7 +469,15 @@ IECoreScene::PrimitivePtr FromMayaMeshConverter::doPrimitiveConversion( MFnMesh
 		MPlug interpolationPlug = fnMesh.findPlug( "ieMeshInterpolation", &st );
 		if ( st )
 		{
+			#if MAYA_API_VERSION >= 201800
```

Comment:
just a style thing, but we'd typically not indent the `#if` stuff, just the regular code

Reply:
Fixed.